### ISSUE TITLE
Profiler: Improve labels for times that round to 0.0ms

### ISSUE THREAD

**Issue by bvaughn** [bvaughn/react-devtools-experimental#259]
A better thing to show would be "<0.1ms"

**Comment by submetu** [comment]
Hi, I tried to check the test app for something that shows a label of 0.0ms.
The least amount of time I was able to find was 0.1ms. 

Can you give me an example that shows a label of 0.0ms?

**Comment by bvaughn** [comment]
Just write a few components that render null 😄 

```js
const Example = () => null;
```

**Comment by submetu** [comment]
Ok i found one in the test app.

**Comment by submetu** [comment]
![image](https://user-images.githubusercontent.com/18688941/57324700-0db72580-7111-11e9-8a70-6eb5930247a4.png)

**Comment by submetu** [comment]
I can take a look at this :)

**Comment by submetu** [comment]
![image](https://user-images.githubusercontent.com/18688941/57328181-65f22580-7119-11e9-9bb4-a864914154c1.png)

Ok I did some stuff.
Doesn't this look a bit odd? 
Or is this what we are aiming for?

**Comment by bvaughn** [comment]
It does look a bit odd, but I think it's less confusing than "0ms of 0ms".

**Comment by submetu** [comment]
Alright. 
To open a PR, should I just create a new branch on my local and push it to the project?

**Comment by bvaughn** [comment]
You should fork this repo, which GitHub makes pretty easy to do by clicking the "Fork" button in the top right.

Then you can add that fork to your copy of this repo, e.g. 
```
git remote add submetu git@github.com:submetu/react-devtools-experimental.git
```
Then you can push your branch to your fork, e.g.
```
git push submetu your-branch
```

Then your Git client should show you a URL in the terminal that you can copy-paste to create a pull request with your branch, e.g.
```
https://github.com/bvaughn/react-devtools-experimental/compare/bvaughn:master...submetu:your-branch
```

**Comment by submetu** [comment]
Alright. Will do that tonight. 
Thanks for the detailed instructions 👍 
Maybe we can add this to the README of the project

**Comment by submetu** [comment]
#262

**Comment by bvaughn** [comment]
Fixed in #262
